tests/fixtures/2_kakarot.py

```diff
@@ -13,7 +13,7 @@ async def eth(starknet: Starknet):
 
 
 @pytest_asyncio.fixture(scope="session")
-async def contract_account_class(starknet: Starknet):
+async def contract_account_class(starknet: Starknet) -> DeclaredClass:
```

Comment:
I was wondering if you could share your suggestion about when adding typing info to code we write in python tests.

Should we always be looking to do that or is it more important at the level of fixtures/helpers?

Reply:
good question. Bottom line is: we could (should ? but in a couple of months we'll probably move to rust) strive to use mypy (python type checking) but it'd be quite a lot of rework. In the meantime, I somehow add these randomly when I want my IDE to understand better what I'm doing.

Reply:
Yeah, as a python noob it took me a while to realize some of the comments like 

```
# type: ignore
```

was conveying information meant for mypy haha. For a while I thought it was just people copy and pasting an old comment.

At readtime, I like the type annotations. I think using them at the level of fixtures/helpers makes sense!